---
title: "Basics of G-quadruplex structures"
description: |
  A quick overview of the (nucleo)bases of G-quadruplex nucleic acids.
author:
  - name: Eric Largy 
    url: https://ericlarg4.github.io/index.html
    affiliation: ARNA
    affiliation_url: https://arna.cnrs.fr/
  - name: Valérie Gabelica 
    url: https://gabelicagroup.wixsite.com/biophyms
    affiliation: ARNA
    affiliation_url: https://arna.cnrs.fr/
  - name: Jean-Louis Mergny 
date: "`r Sys.Date()`"
output:
  distill::distill_article:
    toc: true
    theme: css/theme_distill.css
---

# 1. Definitions

In the following section, minimal definitions of terms specific to the field are proposed to ensure a better consistency in describing G-quadruplex structures across publications.

## 1.1 G-quadruplex

[G-quadruplex]{.ul} (abbrev.: [G4]{.ul}): Secondary structure formed by the association of a minimum of two G-quartets by $\pi$-$\pi$ stacking and coordination to a cation (Figure \@ref(fig:fig1), left). The term G-quadruplex encompasses a variety of conformations differing by their number of G-quartets, loop length and geometry, relative orientation of the G-tract strands, molecularity, and other non-canonical features (e.g. mixed quartets, extended quartets such as hexads or heptads, and additional features including triplets, base pairs or bulges). G-quadruplexes may be formed from nucleic acid having either natural (DNA, RNA) or artificial (e.g. PNA, LNA) backbones.

[Tetraplex]{.ul}: Alternative name for G-quadruplex. Not recommended to use it to avoid confusions with other uses of the word (e.g. tetraplex PCR assay). Tetraplex should not be used to imply a tetra-stranded stoichiometry; preferred wording should be: tetramolecular G-quadruplex.

```{r fig1, echo=FALSE, fig.cap="The G-quadruplex and i-motif cores. Both are formed by stacking of different building blocks, respectively the G-quartet and the hemi-protonated cytosine base pair for the i-motif. The phosphate backbone is shown as a line. Guanines and cytosines are depicted by an orange or blue circle, respectively, and a cuboid", out.width = '100%'}
knitr::include_graphics("img/fig1.png")
```

[Z-G4]{.ul}: Left-handed G-quadruplex, named by analogy with Z-DNA left-handed double helices (DOI: 10.1073/pnas.1418718112).

[i-motifs]{.ul} (or i-DNA) are structures formed at slightly acidic pH by cytosine-rich nucleic acids, characterized by C•CH^+^ base pairs arranged in two parallel duplexes associated head-to-tail via base-pair intercalation (Figure \@ref(fig:fig1), right). Although these double duplexes can be considered as a type of quadruplex structure, they will not be discussed further herein.z\`z

## 1.2 Quartets and derivatives

[G-quartet]{.ul}: Quasi co-planar association of four guanines linked by a network of eight hydrogen bonds on both the Watson-Crick and Hoogsteen faces (i.e. N2H·N7 and N1H·O6) (Figure \@ref(fig:fig2)A). They are also frequently called [quartets]{.ul} or [tetrads]{.ul}, although these terms are less specific and should preferably be avoided.

```{r fig2, echo=FALSE, fig.cap="Chemical structures of the G-quartet (A), G·C·G·C (B; DOI: 10.1093/nar/gkp630), G·G·G·T (C; DOI: 10.1021/bi0355185) and G·G·G·A (D; DOI: 10.1021/bi0355185) mixed quartets, G·G·G G-triplet (E; DOI: 10.1093/nar/gkp1029), and A·G·A mixed triplet (F; DOI: 10.1093/nar/gkp1029).", out.width = '100%'}
knitr::include_graphics("img/fig2.png")
```

[G-tract]{.ul}: Part of a nucleic sequence consisting of a minimum of two consecutive guanines, which may be involved in the formation of *G-quartets* (Figure \@ref(fig:fig3)). G-tracts involved in *G-quartet* formation can be interrupted by [bulges]{.ul} or [snapbacks]{.ul}, and linked to another G-tract by a *loop* (see section 1.3). A minimum of four G-tracts are necessary to form a G-quadruplex, whether they are contained in a single strand or in up to four different strands.

[Stem]{.ul}: Consecutively stacked guanines involved in *G-quartet* formation. There are four stems per *core*.

[Mixed quartet]{.ul}: *Quartet* of bases not entirely formed by guanines, e.g. G·C·G·C, G·G·G·T and G·C·G·A (Figure \@ref(fig:fig2)B---D).

[Triplet]{.ul} (or [triad]{.ul}): By analogy with the *quartet*, quasi co-planar association of three bases. It can be composed of guanines exclusively ([G-triplet]{.ul}; Figure \@ref(fig:fig2)E) or *mixed* (Figure \@ref(fig:fig2)F).

[Base pair]{.ul}: Association of two nucleotides by hydrogen bonding of the bases; not limited to Watson-Crick pairing. Although not typically associated with G-quadruplexes, base pairs can be formed by *loop* and *flanking sequence* nucleotides, and are sometimes stacked upon the *core*, hence contributing to the G4 stabilization.

[Core]{.ul}: Structural ensemble of consecutively stacked *quartets* and their *inner cations* (Figure \@ref(fig:fig1)). Sometimes called subunits in the context of stacked or interlocked multimers (see section 1.8).

[Inner cation]{.ul}: In the particular context of G-quadruplexes, cation bound by the O6 of G-quartets guanines (or related structures, vide supra) (Figures \@ref(fig:fig1) and \@ref(fig:fig2)A). Depending on a number of factors including the cation ionic radius and lone pair attraction, cation-cation repulsion, and possible G-quartet distortion, it may be located anywhere from within a *G-quartet*'s plane to -- more commonly -- midway between two *G-quartets*, as typically observed for potassium (DOI: 10.1007/978-3-319-21756-7).

```{r fig3, echo=FALSE, fig.cap="G-quadruplex features beyond the core (A), loop geometries (B), and snapback (C). The phosphate backbone is shown as a line and guanines are depicted by an orange circle and a cuboid in either syn (dark gray) or anti (light gray) conformation.", out.width = '100%'}
knitr::include_graphics("img/fig3.png")
```

## 1.3 Beyond the G4 core: loops, bulges, and flanking sequences

[Loop]{.ul}: Part of the nucleic acid sequence linking *G-tracts* involved in *G-quartet* formation (Figure \@ref(fig:fig3)B). Loops can adopt different geometries (see section 1.4). Loop nucleotides may be *base-paired*.

[Bulge]{.ul}: Part of a sequence interrupting a *G-tract* involved in *G-quartet* formation, *i.e.* nucleotides linking two guanines implicated in two consecutively stacked *G-quartets* (Figure \@ref(fig:fig3)A).

[Flanking sequence]{.ul}: Nucleotides from the 5'- and 3'-termini of a G-quadruplex positioned before the first or after the last *G-quartet* forming guanine, respectively (Figure 3A). *Flanking sequence* nucleotides may be *base-paired*. Although not part of the *core*, flanking sequences can have a large impact on the topology and multimerization of G-quadruplexes (DOI: 10.1021/jacs.5b13130). Flanking sequences are naturally present in the genomic context.

## 1.4 Loop types

[Propeller]{.ul} (or [chain reversal]{.ul}): *loop* spanning across *G-quartets* to connect guanines from two different *quartets* so that the two linked *G-tracts* are adjacent and share the same 5'-3' polarity (parallel) (Figure \@ref(fig:fig3)B). Parallel loops typically contain 1---2 nucleotides.

[V-shaped]{.ul}: Variant of the *propeller* loop that does not contain any nucleotide, so that the two *G-tracts* are directly connected.

[Lateral]{.ul} or [edgewise]{.ul}: loop connecting two adjacent *G-tracts* of opposite 5'-3' polarity (*antiparallel*) (Figure \@ref(fig:fig3)B). Lateral loops typically contain at least 2---3 nucleotides depending on the *groove* width.

[Diagonal]{.ul}: loop linking diagonally opposed [G-tracts]{.ul} of opposite 5'-3' polarity (antiparallel) (Figure \@ref(fig:fig3)B). Diagonal loops typically contain at least 3 nucleotides.

[Snapback]{.ul}: Loop that connects an external *quartet* to an internal *quartet*., instead of linking two external [quartets]{.ul} as more commonly observed. This means that one *stem* of consecutively stacked guanines is composed of two distinct *G-tracts* (Figure \@ref(fig:fig3)C) (DOI: 10.1021/ja068739h). This implies that the *stem* in question is not composed of a continuous phosphate backbone.

Both *lateral* and *diagonal* loops result in a strand polarity inversion occurring in *antiparallel* and *hybrid* topologies, whereas *propeller* loops preserve the same orientation and are thus found only in *parallel* and *hybrid* topologies (see section 1.6).
